order.go

```diff
@@ -139,20 +139,30 @@ type OrderList struct {
 
 // OrderListParams is the set of parameters that can be used when listing orders.
 type OrderListParams struct {
-	ListParams   `form:"*"`
-	Created      *int64            `form:"created"`
-	CreatedRange *RangeQueryParams `form:"created"`
-	Customer     *string           `form:"customer"`
-	IDs          []*string         `form:"ids"`
-	Status       *string           `form:"status"`
+	ListParams        `form:"*"`
+	Created           *int64                         `form:"created"`
+	CreatedRange      *RangeQueryParams              `form:"created"`
+	Customer          *string                        `form:"customer"`
+	IDs               []*string                      `form:"ids"`
+	Status            *string                        `form:"status"`
+	StatusTransitions *StatusTransitionsFilterParams `form:"status_transitions"`
+}
+
+// StatusTransitionsFilterParams are parameters that can used to filter on status_transition when listing orders.
+type StatusTransitionsFilterParams struct {
```

Comment:
What do you think about calling this `StatusTransitionsParams` for a little more brevity? Also cc @remi-stripe in case you want in on the naming bikeshed ;)

Reply:
The issue with `StatusTransitionsParams` is that if we ever added `status_transitions` as a parameter on the create/update then there's a naming conflict. So either `StatusTransitionsFilterParams` or `StatusTransitionsListParams` is likely safer

Reply:
Ah true! Okay, I'm okay with just leaving as is then. Thanks.

Reply:
Leaving it like this.